repos/fdbt-site/src/pages/viewOperatorGroups.tsx

```diff
@@ -59,7 +59,11 @@ const ViewOperatorGroups = ({
             <div>
                 <ErrorSummary errors={viewOperatorGroupErrors} />
             </div>
-            <div className="govuk-grid-row" data-card-count={operatorGroups.length}>
+            <div
+                className="govuk-grid-row"
+                data-card-count={operatorGroups.length}
+                operator-groups={operatorGroups.map((element) => element.name).toString()}
```

Comment:
Trying to understand why we're adding this, or why we need data-card-count either.

For data-card-count, cant we just check on the page how many card elements there are? or how many id's containing 'operator-group-' there are?

For operator groups, cant we just get the text content of the card? 
![Screenshot 2022-11-22 at 13 12 51](https://user-images.githubusercontent.com/111580926/203322767-2e9e63df-8b3d-47e9-871e-a926ab10a5c2.png)
as per attached image.

Reply:
It would be a longer process to iterate through each card. As for the count I can't find a method that does it without using should and I have just followed what we have done previously to be consistent

Reply:
Okay - we need to be wary about having to add these properties into the UI just to be able to make the UI tests work. In future I think we should exhaust the other solutions before we do this.